src/main/java/com/googlecode/zohhak/api/backend/ParameterCalculatorProvider.java

```diff
@@ -0,0 +1,16 @@
+package com.googlecode.zohhak.api.backend;
+
+import com.googlecode.zohhak.internal.IoCContainer;
+import com.googlecode.zohhak.internal.coercing.DefaultParameterCoercerFactory;
+
+public class ParameterCalculatorProvider {
+
+    public ParameterCalculator getExecutor() {
+        return getExecutor(new DefaultParameterCoercerFactory());
+    }
+
+    public ParameterCalculator getExecutor(ParameterCoercerFactory parameterCoercerFactory) {
```

Comment:
There does not need to be an implementation of `ParameterCoercerFactory`, only an interface that can be implemented - see here for my usage:
https://github.com/theangrydev/yatspec-zohhak-plugin/blob/master/src/main/java/io/github/theangrydev/yatspeczohhakplugin/ZohhakParameterResolver.java#L28